core/src/test/java/org/bptlab/cepta/sum-of-delay-at-station/SumOfDelayAtStationTests.java

```diff
@@ -59,4 +65,37 @@ public void TestSumOfDelaysAtStation() throws IOException {
         
         Assert.assertTrue(pass);
     }
+
+    @Test
+    public void TestSumOfDelaysAtStationWithLiveTrainData() throws IOException {
```

Comment:
What exactly does this test? 
Something like TestSumOfDelayEqualsSumOfDelayInput could be better maybe

Reply:
This tests proves that the generics works on LiveTrainData as well on TrainDelayNotification